cartridges/int_algolia/cartridge/scripts/algolia/helper/jobHelper.js

```diff
@@ -533,6 +559,7 @@ function updateCPObjectFromXML(xmlFile, changedProducts, resourceType) {
             xmlStreamReader.close();
         };
     } catch (error) {
+        var e = error;
```

Comment:
What's this for?

Reply:
Indeed, this is a leftover. Let me remove it.